### ISSUE TITLE
how add additional plugin or app like Mega.nz cloud or Terminal to the kod  

### ISSUE THREAD

**Issue by codesiddhant** [kalcaddle/KodExplorer#384]
is it possible to add any application or plugin by own to this project?

**Comment by simon511000** [comment]
Absolutely, KodExplorer has a good API for plugins.
BUT, the documentation is in Chinese, so I advise you:
- Read the other KodExplorer plugins
- Try to understand the documentation with Google Translate (good luck lol)

**Comment by codesiddhant** [comment]
Still trying hard.
and I still haven't received any response from the author.
I mailed him 3 days back in English.
later I also mailed him 😄 in Chinese but again no luck

**Comment by simon511000** [comment]
1 year ago I tried to read and understand the KodExplorer source code.
Reading PHP helped me a lot,
Unfortunately, JS files are compiled. Therefore, to understand the API, I created console.log on almost every object and I can assure you that the API is very complete.

I also tried to read the documentation with Google Translate, which also helped me a lot.
And finally, trying to understand other plugins is essential.

It's a shame to take so long to understand something that is very well documented ...

KodExplorer is great, but there is almost no English community and @kalcaddle does nothing to improve that.
The back end of KodExplorer is not even open-source!

KodExplorer has a lot of problems and that's why I stopped working with it.
